lms/templates/dashboard/_dashboard_certificate_information.html

```diff
@@ -54,6 +54,12 @@
       <a class="btn" href="${cert_status['download_url']}"
          title="${_('This link will open/download a PDF document')}">
          ${_("Download Your {cert_name_short} (PDF)").format(cert_name_short=cert_name_short,)}</a></li>
+
+      <li class="action action-certificate">
+      <a class="btn"target="_blank" href="${cert_status['linked_in_url']}"
+         title="${_('Add to LinkedIn Profile')}">
+         ${_("Add Certificate to LinkedIn.")}</a></li>
```

Comment:
If the configuration isn't provided, this button should be hidden.  I'd recommend doing it this way:

1) In `views.py`, check whether `LinkedInUrlConfiguration` is enabled.  (All config models have this property.)
2) If so, include `linked_in_url` in the `cert_status` dict.
3) In the template, hide the button unless `linked_in_url` is provided.